src/webui/www/private/shareratio.html

```diff
@@ -182,6 +189,16 @@
             <label id="inactiveMinutesLabel" for="setInactiveMinutes">QBT_TR(inactive minutes)QBT_TR[CONTEXT=UpDownRatioDialog]</label>
             <input type="number" id="inactiveMinutes" value="0" step="1" min="0" class="shareLimitInput" aria-labelledby="inactiveMinutesLabel">
         </div>
+        <div style="margin-left: 40px; margin-bottom: 5px;">
+            <label id="actionListLabel" for="action">QBT_TR(Action when the limit is reached)QBT_TR[CONTEXT=UpDownRatioDialog]</label>
+            <select id="action" aria-labelledby="actionListLabel">
```

Comment:
This `id` is a bit too generic. I would suggest to give it a bit more precise name. For example `limitActions` or `limitReachedActions`.